protocol-units/da-sequencer/node/src/error.rs

```diff
@@ -3,4 +3,6 @@
 pub enum DaSequencerError {
 	#[error("Error during storage access: {0}")]
 	StorageAccess(String),
+	#[error("Generic error: {0}")]
+	Generic(String),
```

Comment:
I think each variant to be specific to some error case.

Reply:
Yeah, am aware. I thought it could be a nice follow up PR. As this PR needs to go in for others. It can be a refactor followup.